source/ed/connectors/fusio_parser.cpp

```diff
@@ -280,7 +280,7 @@ void TransfersFusioHandler::fill_stop_point_connection(ed::types::StopPointConne
 
 void StopTimeFusioHandler::init(Data& data) {
     StopTimeGtfsHandler::init(data);
-    itl_c = csv.get_pos_col("stop_times_itl");
```

Comment:
I know it's quite bothersome, but I think it would be better to add some itl to the ntfs fixture, just to be sure this won't happen again :wink: 

`navitia/fixtures/ed/ntfs/` and `navitia/source/ed/tests/fusioparser_test.cpp`